templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="font-weight: bold; color: red;">
+      Invalid credentials. Try again!
+    </p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}

```

Comment:
The form is rendered using `form.as_p`, which is a simple and effective way to display form fields with paragraph tags.